### PR TITLE
Add support for PBXShellScriptBuildPhase

### PR DESCRIPTION
**Pull request by sundbry**
This implements PBXShellScriptBuildPhase inside project.addBuildPhase. 

Resolves issue #85 and upgrades version to 0.9.0

### CODE REVIEW

File: lib/pbxProject.js [hunk 1]
```diff
@@ -1523,6 +1525,18 @@ function pbxCopyFilesBuildPhaseObj(obj, folderType, subfolderPath, phaseName){
     obj.name = '"' + phaseName + '"';
     obj.dstPath = subfolderPath || '""';
     obj.dstSubfolderSpec = SUBFOLDERSPEC_BY_DESTINATION[DESTINATION_BY_TARGETTYPE[folderType]];
+    console.log("pbxCopyFilesBuildPhase");
+    console.log(obj);
+
```
Comment (imhotep) [inline lib/pbxProject.js]: Hi, remove console.log()s and add unit tests please

**Comment by sundbry** [comment]
@imhotep There we go. Lots of tests for this project. 👍

**Comment by sundbry** [comment]
I opened a separate pr in #108 without the path name bugfix, to lower the risk of merging this all in.

**Comment by chrismowbray** [comment]
Hey sundbry.

I added an isssue (https://github.com/alunny/node-xcode/issues/125) in relation to these changes. Would be great if there was a way to remove a build phase also.

**Comment by sundbry** [comment]
@chrismowbray I can see why you'd want that, especially for making scripts idempotent etc. I can't promise I will get around to implementing it any time soon though.

**Pull request closed without merging**